src/universal/modules/teamDashboard/containers/TeamSettings/TeamSettingsContainer.js

```diff
@@ -66,6 +70,7 @@ const TeamSettingsContainer = (props) => {
   return (
     <TeamSettings
       dispatch={dispatch}
+      router={router}
```

Comment:
same as above, let's put the HOC _just_ on the component that needs the router instead of the parent.